features.json

```diff
@@ -6,56 +6,106 @@
 			"url": "https://github.com/WebAssembly/JS-BigInt-integration",
 			"phase": 5
 		},
+		"branchHinting": {
+			"description": "Branch Hinting",
+			"url": "https://github.com/WebAssembly/branch-hinting/blob/master/proposals/branch-hinting/Overview.md",
+			"phase": 5
+		},
 		"bulkMemory": {
 			"description": "Bulk memory operations",
 			"url": "https://github.com/WebAssembly/bulk-memory-operations/blob/master/proposals/bulk-memory-operations/Overview.md",
 			"phase": 5
 		},
+		"compilationHints": {
+			"description": "Compilation Hints",
+			"url": "https://github.com/WebAssembly/compilation-hints/blob/main/proposals/compilation-hints/Overview.md",
+			"phase": 1
+		},
+		"componentModel": {
+			"description": "Component Model",
+			"url": "https://github.com/WebAssembly/component-model",
+			"phase": 1
+		},
+		"customAnnotationSyntaxInTheTextFormat": {
+			"description": "Custom Text Format Annotations",
+			"url": "https://github.com/WebAssembly/annotations/blob/main/proposals/annotations/Overview.md",
+			"phase": 5
+		},
 		"customPageSizes": {
-			"description": "Custom page sizes",
+			"description": "Custom Page Sizes",
```

Comment:
Most titles on the table currently use Title Case, but there are still a few sentence cases mixed in, like "ESM integration" below. This may be a good opportunity to unify them.

Reply:
Most (but not all) of the proposals listed before this PR only had the first word capitalized. https://github.com/WebAssembly/proposals mostly (but not fully) uses title case. I don't have a strong opinion about which, but maybe just switch the missing few here to title case for consistency would be good.

Reply:
I updated it to use title case everywhere.